src/write.rs

```diff
@@ -176,6 +172,9 @@ where
                         thread::yield_now();
                     }
 
+                    #[cfg(loom)]
+                    loom::thread::yield_now();
```

Comment:
I think we should also make the `if` just above be `cfg(not(loom))`

Reply:
Makes sense to me! Updated in 4342b2262507916c2f437a8693137a6d0bed0635. Not sure if this would like better with like an `if cfg!(loom) { } else { }` - lemme know your preference there (or maybe that's not possible?). Also, this now causes some unused variable complaints when running with the loom configuration. I assume that's fine but I can chase those down if you want!

Reply:
I think I'd just surround the original `if else` in an `if cfg!(loom)`, and then not place the `#[cfg(loom)]` in an `else` block but leave it like you have here.

Reply:
Hmmm, sorry, confused again :sweat_smile: do you mean like:
```rust
if !cfg!(loom) {
  if iter != 20 { ... } else { ... }
}

#[cfg(loom)]
loom::thread::yield_now();
```

Reply:
Yes indeed. That should get rid of the "unused" warnings :)

Reply:
Ah, clever! Added in 62a4de0221322e5bdf6f6ad71df05d8f113bba22